.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+# Reference used: https://circleci.com/docs/2.0/language-android/#docker-images
+
+version: 2
+
+jobs:
+  build:
+    working_directory: ~/code
+
+    docker:
+      - image: circleci/android:api-23-alpha
+
+    environment:
+      # https://discuss.circleci.com/t/circle-ci-v2-and-android-memory-issues/11207/9
```

Comment:
Since the link has a bunch of things to try, I'd add a comment that says this is the min change that fixed the problem.